tests/course/test_course_pages.py

```diff
@@ -71,3 +71,31 @@ def test_add_course(self, authorized_client):
         assert course_creation_response.status_code == 200
         assert 'courses.html' in course_creation_response.templates[0].name
         assert TeacherCourse.objects.filter(**teacher_course).exists()
+
+
+@pytest.mark.django_db
+class TestCoursePageView:
+
+    def test_coures_page_authorized(self, authorized_client, persist_course):
+        course_response = authorized_client.get('/course/' + str(persist_course.course_id))
+        assert course_response.status_code == 200
+        assert 'course_page.html' in course_response.templates[0].name
```

Comment:
put it into a variable to make it more readable, for example `expected_template_name`